File: rest/usage-records/list-get-example-4/list-get-example-4.4.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
client = Twilio::REST::Client.new(account_sid, auth_token)

# Loop over records and print out a property for each one
client.account.usage.records.daily.list(
  category: 'calls-inbound',
  start_date: '2012-09-01',
  end_date: '2012-09-30'
).each do |record|
  puts record.price
end

File: rest/notification/list-get-example-4/list-get-example-4.4.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; 
$token = "your_auth_token"; 
$client = new Services_Twilio($sid, $token);

// Loop over the list of notifications and echo a property for each one
foreach ($client->account->notifications->getIterator(0, 50, array(
        "MessageDate>" => "2009-07-06",
    "MessageDate<" => "2009-07-08",
    "Log" => "1"
    )) as $notification
) {
    echo $notification->request_url;
}
